src/pages/Product/Product.js

```diff
@@ -1,7 +1,244 @@
-import React from 'react';
+import React, { useEffect, useState, useCallback } from 'react';
+import ProductList from './ProductList';
+import { AiOutlinePlus } from 'react-icons/ai';
+import './Product.scss';
+import Footer from '../../components/Footer/Footer';
 
 const Product = () => {
-  return <div>Product</div>;
+  const [products, setProducts] = useState([]);
+  const [showSize, setShowSize] = useState(false);
+  const [showColor, setShowColor] = useState(false);
+  const [showAct, setShowAct] = useState(false);
+  const [showCategory, setShowCategory] = useState(false);
+  const [pCategory, setpCategory] = useState('productData');
+
+  useEffect(() => {
+    const query = pCategory === 'productData' ? 'productData' : `${pCategory}`;
+    fetch(`data/${query}.json`)
+      .then(res => res.json())
+      .then(data => {
+        setProducts(data);
+      });
+  }, [pCategory]);
+
+  const sizeToggle = () => setShowSize(!showSize);
+  const colorToggle = () => setShowColor(!showColor);
+  const actToggle = () => setShowAct(!showAct);
+  const categoryToggle = () => setShowCategory(!showCategory);
+  const onSelect = useCallback(pCategory => setpCategory(pCategory), []);
+  return (
+    <>
+      <section className="product">
+        <div className="productWrapper">
+          <div className="productLeft">
+            <div className="productLeftName">
+              <h1>Men's Clothes</h1>
+            </div>
+            <div className="filterContainer">
+              <button className="filterDetail">red</button>
+              <button className="filterDetail">S</button>
+            </div>
+            <div className="filterBox">
+              <div className="categoryBox">
+                <div className="categoryTitle">
+                  <h2>Category</h2>
+                  <AiOutlinePlus className="plus" onClick={categoryToggle} />
+                </div>
+                <div className="categoryText">
+                  {showCategory &&
+                    subCategory.map(c => {
+                      const { name, category, id } = c;
```

Comment:
파라미터명도 축약어 사용어는 지양해주세요!
```suggestion
                    subCategory.map({ name, category, id } => {
```
데이터가 많지 않다면 파라미터에서 구조분해 할당해주시는게 더 깔끔할 것 같습니다!

Reply:
수정 완료했습니다!